egs/wsj/s5/steps/nnet3/dump_bottleneck_features.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+#           2016  Pegah Ghahremani
+# Apache 2.0
+# This script dumps bottleneck feature for model trained using nnet3.
+
+# Begin configuration section.
+feat_type=
+stage=1
+nj=4
+cmd=run.pl
+use_gpu=false
+bnf_name=renorm4
```

Comment:
Bnf_name seems to be a very critical parameter in this script. I think it is unwise to assign it by default, as we frequently vary the neural network architecture in our models.